### DISCUSSION TITLE
建议功能：自定义音量快捷键变动声音幅度

### DISCUSSION THREAD

**Discussion by CuewarsTaner** [the1812/Bilibili-Evolved#2594] [功能建议 / Ideas]
默认按↑/↓键是+/-10音量，可以不可自定义一个幅度，例如改成+/-5。
感谢作者！

**Comment by Geeyun-JY3** [comment]
关联：#718

**Comment by LegendAdmin** [comment]
没想到1年过去了依然没出这个功能🤣

**Comment by Godwin9527** [comment]
同求

**Comment by HornyPrivateGamer** [comment]
求大佬更新下

**Comment by Godwin9527** [comment]
https://h5player.anzz.top/zh/
你可以尝试使用这个脚本来解决问题，对几乎所有视频都有效，且自定义程度也更好。

**Reply by Geeyun-JY3** [reply]
以防你不知道：https://github.com/the1812/Bilibili-Evolved/discussions/5043#discussioncomment-11337557
